dockers/base-conda/Dockerfile

```diff
@@ -73,9 +74,10 @@ COPY environment.yml environment.yml
 # conda init
 RUN \
     conda update -n base -c defaults conda && \
+    CUDA_VERSION_MM=$(python -c "print('.'.join('$CUDA_VERSION'.split('.')[:2]))") && \
     conda create -y --name $CONDA_ENV \
-      python=${PYTHON_VERSION} pytorch=${PYTORCH_VERSION} torchvision torchtext cudatoolkit=${CUDA_VERSION} \
-      -c nvidia -c pytorch -c pytorch-test -c pytorch-nightly && \
```

Comment:
Why is this removed? it serves for sourcing future test or nightly versions...

Reply:
We're pinning the version of pytorch anyway, so `pytorch-nightly` is definitely never going to be used (it has versions like `1.13.0.dev`. `pytorch-test` seems like a duplicate of `pytorch` channel and they looked obsolete when pinning versions anyway.

Reply:
If you want, I can add them back, but then the building of the images will take another hour